main.scss

```diff
@@ -13,6 +13,8 @@
 	@import "src/scss/print";
 	@import "src/scss/search";
 	@import "src/scss/tools";
+	@import "src/scss/themes/dark-on-light";
```

Comment:
Cool, makes sense. Initial reasoning is perfectly sensible. I wonder if now that there's a default which is dark and the theme starts `dark-...` it could be potentially confusing?